components/eam/src/chemistry/yaml/aero_model/modal_aero_bcscavcoef_get_output_ts_355.py

```diff
@@ -0,0 +1,22 @@
+# Data at [lat=  -13.2292708024991      , lon=   139.817062054960      , k=          53 , time step=         355 ] 
+# This file was generated by E3SM.
+
+from math import nan as nan, inf as inf
+
+# Object is just a dynamic container that stores input/output data.
+class Object(object):
+    pass
+# Settings are stored here.
+settings = Object()
+# Input is stored here.
+input = Object()
+input.dt = [    0.00, ]
+input.imode=[[         1],]
+input.ncol=[[         4],]
+input.isprx=[[         T],]
+input.dgn_awet=[[ 0.1522452427E-06, 0.4627511718E-07, 0.1886138608E-05, 0.9911179496E-07,],]
+input.dgnum_amode=[[ 0.1100000000E-06, 0.2600000000E-07, 0.2000000000E-05, 0.5000000058E-07,],]
+# Output data is stored here.
+output = Object()
+output.scavcoefnum=[[ 0.6254861968E-03],]
```

Comment:
Can we save `scavcoefnum` and `scavcoefvol` for the ncol?. In this case this will be a vector of 4.

Reply:
yes we can save the variables for all ncols. Just wondering, do you want ncols only for this subroutine or for all tests in the future?

Reply:
only for this subroutine.Thanks!

Reply:
test files updated.